packages/webpack-cli/lib/utils/logger.js

```diff
@@ -2,10 +2,26 @@ const utils = require("./index");
 const util = require("util");
 
 module.exports = {
-  error: (val) => console.error(`[webpack-cli] ${utils.colors.red(util.format(val))}`),
-  warn: (val) => console.warn(`[webpack-cli] ${utils.colors.yellow(val)}`),
-  info: (val) => console.info(`[webpack-cli] ${utils.colors.cyan(val)}`),
-  success: (val) => console.log(`[webpack-cli] ${utils.colors.green(val)}`),
+  error: (val) => {
+    const { red } = utils.colors.createColors({ useColor: utils.colors.isColorSupported });
+
+    return console.error(`[webpack-cli] ${red(util.format(val))}`);
+  },
+  warn: (val) => {
+    const { yellow } = utils.colors.createColors({ useColor: utils.colors.isColorSupported });
+
+    return console.warn(`[webpack-cli] ${yellow(val)}`);
+  },
+  info: (val) => {
+    const { cyan } = utils.colors.createColors({ useColor: utils.colors.isColorSupported });
+
+    return console.info(`[webpack-cli] ${cyan(val)}`);
+  },
+  success: (val) => {
+    const { green } = utils.colors.createColors({ useColor: utils.colors.isColorSupported });
+
+    return console.log(`[webpack-cli] ${green(val)}`);
+  },
```

Comment:
Very bad idea

Reply:
This was now required for `--no-color` support.